platform/pulpcore/tasking/tasks.py

```diff
@@ -243,14 +245,25 @@ def apply_async(self, tags=[], group_id=None, args=None, kwargs=None, **options)
 
     def __call__(self, *args, **kwargs):
         """
-        Set the :class:`pulpcore.app.models.Task` object in the running state and log some output.
+        Init code for our task.
+
+        Set the :class:`pulpcore.app.models.Task` object in the running state,
+        associate the worker, and log a message.
 
         Skip the status updating if the task is called synchronously.
```

Comment:
Does it not do it?  Not in the common case, but I thought we still need this code for other scenarios.